nasa-gibs/worldview
# WV-3100: Add NOAA-21 orbit tracks

## Description

Fixes #WV-3100 .

Add NOAA-21 Ascending and Descending Orbit Tracks

## How To Test

1. `git checkout wv-3100-noaa21-orbit-tracks`
2. `npm run build && npm start`
3. Open with [these URL parameters](http://localhost:3000/?v=-221.27401006694546,-108.96476223389257,208.30230173663418,99.88345572813535&l=OrbitTracks_NOAA-21_Descending,OrbitTracks_NOAA-21_Ascending,Land_Mask(hidden,opacity=0.8),Land_Water_Map(opacity=0.8)&lg=true&t=2024-04-08-T14%3A18%3A06Z)
4. Check that the NOAA-21 orbit tracks load on map, that the points are clickable when you zoom in, that you can find the orbit tracks in the Layer Picker


## PR Submission Checklist

This is simply a reminder of what we are going to look for before merging your code.

- I have read the [CONTRIBUTING](https://github.com/nasa-gibs/worldview/blob/main/.github/CONTRIBUTING.md) doc
- I have added necessary documentation (if applicable)
- I have added tests that prove my fix is effective or that my feature works (if applicable)
- Any dependent changes have been merged and published in downstream modules (if applicable)

## Merging

Please use the `squash and merge` commit method unless each commit in your branch is vital to the commit history of main.

@nasa-gibs/worldview
